src/Bex/Behat/ScreenshotExtension/Listener/ScreenshotListener.php

```diff
@@ -107,7 +114,10 @@ public function saveScreenshot(AfterScenarioTested $event)
      */
     private function shouldTakeScreenshot(AfterTested $event)
     {
-        return $event->getTestResult()->getResultCode() !== TestResult::SKIPPED;
```

Comment:
`shouldSaveScreenshot()` needs to be simplified, because the code is duplicated now between `shouldSaveScreenshot()` and `shouldTakeScreenshot()`.